drivers/hmis/app/graphql/types/hmis_schema/client.rb

```diff
@@ -212,6 +212,13 @@ def user
       load_ar_association(object, :user)
     end
 
+    def activity_log_field_name(field_name)
+      case field_name
+      when 'ssn', 'dob', 'disabilities', 'disability_groups'
+        field_name
+      end
+    end
+
```

Comment:
Can we add the below to the Enrollment type? I tried it locally and it wasn't logging even on the Enrollment dash, maybe I was missing something.

```
    def activity_log_field_name(field_name)
      case field_name
      when 'disabling_condition'
        field_name
      end
    end
```